.travis.yml

```diff
@@ -14,3 +14,4 @@ addons:
 script:
   - coverage run --source=hc manage.py test
 after_success: coveralls
+cache: pip
```

Comment:
should be caching the pip dependencies for faster builds, the actual test runs in a few seconds, but up to 2 mins are spent just re-installing stuff all the time